platform-modules/mimetype-manager/pom.xml

```diff
@@ -30,7 +35,7 @@
         <dependency>
             <groupId>org.sunbird</groupId>
             <artifactId>cloud-store-sdk</artifactId>
-            <version>1.2.8</version>
+            <version>1.3.0</version>
         </dependency>
```

Comment:
Same dependency repeated multiple times. Please remove duplicate dependencies.

Reply:
Removed the duplicate dependencies.